### DISCUSSION TITLE
ggwave bindings for rust

### DISCUSSION THREAD

**Discussion by denizsincar29** [ggerganov/ggwave#88] [Ideas]
Hello. Are you planning to add ggwave rust bindings? I'm thinking to make a simple command line program for sending and recieving ggwave data, but python programs are so big in executable size...

**Comment by ggerganov** [comment]
Nope, I don't have experience with rust.
If someone is willing to contribute, then we can add them to the project.

**Comment by denizsincar29** [comment]
Georgy. If someone knows how to use bindgen, then they can make rust bindings.

**Comment by Thoxy67** [comment]
I have made a simple FFI Rust binding for ggwave.

Here is the repo : [ggwave-rs](https://github.com/Thoxy67/ggwave-rs)
